docker/development-psql/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM postgres:10.12
```

Comment:
Variants exist now for postgres 10 and 9.  Timur and magma use postgres 10 right now just so that they have access to the db_seeds generated from the vm (which used postgres 10). 

We'll, in the future, need to carefully upgrade  postgres across all apps to 10.